File: sample/src/main/java/com/sha/formvalidatorsample/util/SnackBarUtil.java
package com.sha.formvalidatorsample.util;

import android.os.Handler;
import android.view.View;
import android.widget.TextView;

import com.google.android.material.snackbar.Snackbar;
import com.sha.formvalidatorsample.R;

public class SnackBarUtil {

    public static void gotIt(final View view, String text) {
        new Handler().postDelayed(() -> multilineSnackbar(
                Snackbar.make(
                        view, text, Snackbar.LENGTH_INDEFINITE)
                        .setAction("Got it", view1 -> {

                        })
        ).show(), 200);
    }

    private static Snackbar multilineSnackbar(Snackbar snackbar) {
        TextView textView = snackbar.getView().findViewById(R.id.snackbar_text);
        textView.setMaxLines(5);
        return snackbar;
    }
}

File: sample/src/main/java/com/sha/formvalidatorsample/classic/util/SnackBarUtil.kt
package com.sha.formvalidatorsample.classic.util

import android.os.Handler
import android.view.View
import android.widget.TextView

import com.google.android.material.snackbar.Snackbar
import com.sha.formvalidatorsample.R

object SnackBarUtil {

    fun gotIt(view: View, text: String) {
        Handler().postDelayed({
            multilineSnackbar(
                    Snackbar.make(
                            view, text, Snackbar.LENGTH_INDEFINITE)
                            .setAction("Got it") {

                            }
            ).show()
        }, 200)
    }

    private fun multilineSnackbar(snackbar: Snackbar): Snackbar {
        val textView = snackbar.view.findViewById<TextView>(R.id.snackbar_text)
        textView.maxLines = 5
        return snackbar
    }
}
